src/lib/containers/HelpPopover.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { MarkdownPopover } from './MarkdownPopover'
+import { Placement } from 'react-bootstrap/esm/Overlay'
+import { HelpOutlineTwoTone } from '@material-ui/icons'
+
+export type HelpPopoverProps = {
+  markdownText: string
+  helpUrl?: string
+  placement?: Placement
+  showCloseButton?: boolean
+}
+
+export const HelpPopover: React.FunctionComponent<HelpPopoverProps> = ({
+  markdownText,
+  helpUrl,
+  placement = "bottom",
+  showCloseButton = true,
+}: HelpPopoverProps) => {
+  
+  const actionButtonConfig = helpUrl ? {
+    content: <>More info</>,
+    closePopoverOnClick: true,
+    onClick: () => window.open(helpUrl, '_blank'),
+    variant: 'primary'
+  } : undefined
+  return (
+    <>
+      <MarkdownPopover
```

Comment:
I see this being implemented in a few ways across the codebase, like to standardize and make reusable.